spec/board_spec.rb

```diff
@@ -0,0 +1,35 @@
+require './lib/board.rb'
+require './lib/player.rb'
+
+RSpec.describe Board do
+
+@player1 =Player.new('❌')
```

Comment:
kindly use `let`, [here](https://medium.com/@tomkadwill/all-about-rspec-let-a3b642e08d39) is a good place to start